js/views/pauseScreen.js

```diff
@@ -9,30 +9,35 @@ var React = require('react'),
     UpNextPanel = require('../components/upNextPanel'),
     TextTrackPanel = require('../components/textTrackPanel'),
     Watermark = require('../components/watermark'),
-    ResizeMixin = require('../mixins/resizeMixin'),
     Icon = require('../components/icon'),
     SkipControls = require('../components/skipControls'),
     Utils = require('../components/utils'),
     CONSTANTS = require('./../constants/constants'),
-    AnimateMixin = require('../mixins/animateMixin'),
     ViewControlsVr = require('../components/viewControlsVr');
-var createReactClass = require('create-react-class');
-var PropTypes = require('prop-types');
 
-var PauseScreen = createReactClass({
-  mixins: [ResizeMixin, AnimateMixin],
-
-  getInitialState: function() {
-    return {
+class PauseScreen extends React.Component{
+  //Alex: When transitioning to a class, I've removed the usage of mixins since they have been
```

Comment:
I think this might be a good candidate for an HOC

Reply:
I agree too. I should've mentioned this might be temporary until we come up with a design solution, but an alternative in the meanwhile is just replicate our mixins as HOCs. What do you think?

Reply:
Sure, sounds good to me